### ISSUE TITLE
Initial discussion about seismology

### ISSUE THREAD

**Issue by core-man** [seismo-learn/seismology#53]
**地震学参考书的目标：**

介绍地震学基础。

**内容：**

1. 震源：理论和正演算法
2. 地震波：理论和正演算法
3. 地球内部结构
4. 地震仪器: 整理、翻译、重排地震仪器基础
    - New Manual of Seismological Observatory Practice：第 4、5、6、7、10 等章
    - Instrumentation in Earthquake Seismology
    - Of Poles and Zeros - Fundamentals of Digital Seismology
5. 地震学数据
  - 下载
  - 处理
  - 分析
6. 地震学反演：理论和反演算法
  - 震源
  - 成像
  - 深部

**主页：** https://seismo-learn.org/seismology/
**源码：** https://github.com/seismo-learn/seismology

seismo-learn/maintenance#5

**Comment by core-man** [comment]
@seisman 
Since we begin to work on software, seismology101, and seismology, I think we should have a content draft for seismology101 and seismology. So, we won't have to revise a lot later.

**Comment by core-man** [comment]
A coarse content of seismology:
- seismic source
- seismic wave
- seismograph
- seismic inversion

----

The content of [Introduction to Seismology (2019)](https://www.cambridge.org/us/academic/subjects/earth-and-environmental-science/solid-earth-geophysics/introduction-seismology-3rd-edition?format=HB&isbn=9781316635742)
1. Introduction
2. Stress and strain
3. The seismic wave equation
4. Ray theory: travel times
5. Inversion of travel time data
6. Ray theory: amplitude and phase
7. Reflection seismology and related topics
8. Surface waves and normal modes
9. Earthquakes and source theory
10. Earthquake prediction
11. Seismometers and seismographs
12. Earth noise
13. Anisotropy

----

The content of [Modern Global Seismology](https://www.elsevier.com/books/modern-global-seismology/lay/978-0-12-732870-6)
1. Introduction
2. Elasticity and seismic waves
3. Body waves and ray theory
4. surface waves and free oscillations
5. Seismometry
6. Seismogram interpretation
7. Determination of Earth structure
8. Seismic sources
9. Earthquake kinematics and dynamics
10. Seismic waveform modeling
11. Seismotectonics

----

The content of [An Introduction to Seismology, Earthquakes, and Earth Structure](https://www.wiley.com/en-us/An+Introduction+to+Seismology%2C+Earthquakes%2C+and+Earth+Structure-p-9780865420786)

1. Introduction
2. Basic Seismological Theory
3. Seismology and Earth Structure
4. Earthquakes
5. Seismology and Plate Tectonics
6. Seismograms as Signals
7. Inverse Problems

----

The content of [Quantitative Seismology (2002)](https://www.ldeo.columbia.edu/~richards/Aki_Richards.html)
1. Introduction
2. Basic Theorems in Dynamic Elasticity
3. Representation of Seismic Sources
4. Elastic Waves from a Point Dislocation Source
5. Plane Waves in Homogeneous Media and Their Reflection and Transmission at a Plane Boundary
6. Reflection and Refraction of Spherical Waves; Lamb's Problem
7. Surface Waves in a Vertically Heterogeneous Medium
8. Free Oscillations of the Earth
9. Body Waves in Media with Depth-Dependent Properties
10. The Seismic Source: Kinematics
11. The Seismic Source: Dynamics
12. Principles of Seismometry

The 1980 version is a little different:

10. Principles of Seismometry (last chapter of Vol 1)
11. Analysis of Seismological data (first chapter of Vol 1)
12. Inverse Problems in Seismology
13. Seismic Waves in 3D inhomogeneous media
14. The Seismic Source: Kinematics
15. The Seismic Source: Dynamics

----

The content of [Theoretical Global Seismology](https://press.princeton.edu/books/paperback/9780691001241/theoretical-global-seismology)
1. Historical Introduction

### Part I Foundations

2. Continuum Mechanics
3. Equations of Motion
4. Normal Modes
5. Seismic Source Representation
6. Anelasticity and Attenuation
7. Rayleigh-Ritz Method

### Part II The Spherical Earth

8. Spheroidal and Toroidal Oscillations
9. Elastic and Anelastic Perturbations
10. Synthetic Seismograms
11. Love and Rayleigh Waves
12. Mode-Ray Duality

### Part III The Aspherical Earth

13. Perturbation Theory
14. Mode Splitting and Coupling
15. Body-Wave Ray Theory
16. Surface-Wave JWKB Theory

**Comment by core-man** [comment]
The course syllabus in 2018 by Zhao Li:

<img width="653" alt="syllabus-2018-ZhaoLi" src="https://user-images.githubusercontent.com/50591376/103367442-0e7ba700-4b00-11eb-980d-1f959f2bd8fd.png">

**Comment by seisman** [comment]
I'm not sure if we really want to include topics like strain and stress. They're important but really boring.

**Comment by core-man** [comment]
I think stress and strain is just in foundation part. they can be added separately any time later. I don't want to work on it now either.

**Comment by core-man** [comment]
I prefer Peter and Aki's book contents.

**Comment by seisman** [comment]
Yes, we can work on the interesting parts first. I actually don't mind changing the file names or chapter/section structures later.

**Comment by core-man** [comment]
okay. now, we first have to name a chapter to include phase name and tau-p sections. how about `射线理论：走时`

**Comment by seisman** [comment]
We can determine the chapter name when someone wants to start that section.

**Comment by core-man** [comment]
okay~

**Comment by core-man** [comment]
@seisman FYI, I transferred this issue from `maintenance` to `seismology`.
